### PR TITLE
Feature/zeus 4238 mux plugin

### PR DESCRIPTION
**Pull request by KharchenkoAlex**
Added Mux analytics

Task: https://pa-media-group.atlassian.net/browse/ZEUS-4238

### CODE REVIEW

File: playback-sdk-android/src/main/java/com/streamamg/PlaybackSDKManager.kt [hunk 1]
```diff
@@ -93,18 +95,21 @@ object PlaybackSDKManager {
...
      * Composable function that loads and renders the player UI.
      * @param entryID The ID of the entry.
+     * @param viewerId From CloudPay (CustomerID)
      * @param authorizationToken The authorization token.
      * @param onError Callback for handling errors. Default is null.
      */
     @Composable
     fun loadPlayer(
         entryID: String,
+        viewerId: String?,
```
[the remark this answers is not in the document]
Comment (KharchenkoAlex) [reply playback-sdk-android/src/main/java/com/streamamg/PlaybackSDKManager.kt:105]: This is CloudPay session for user and, as for me, the session loaded inside app, as we have in BlazeApp and then this is passed into SDK

**Review by StefanoStream — COMMENTED** [review]
Commented

File: playback-sdk-android/src/main/java/com/streamamg/player/plugin/bitmovin/BitmovinVideoPlayerPlugin.kt [hunk 2]
```diff
@@ -165,6 +183,27 @@ class BitmovinVideoPlayerPlugin : VideoPlayerPlugin {
...
+        context: Context,
+        videoTitle: String,
+        videoId: String?,
+        viewerId: String?
+    ) {
+        if (videoId != null && viewerId != null) {
+            PlaybackSDKManager.muxEnvKey?.let { envKey ->
+                MuxAnalyticsManager.track(
+                    context,
+                    playerView!!,
```
[the remark this answers is not in the document]
Comment (KharchenkoAlex) [reply playback-sdk-android/src/main/java/com/streamamg/player/plugin/bitmovin/BitmovinVideoPlayerPlugin.kt:198]: Sure, already uploaded changes

**Review by StefanoStream — APPROVED** [review]

File: playback-sdk-android/src/main/java/com/streamamg/player/plugin/bitmovin/BitmovinVideoPlayerPlugin.kt [hunk 3]
```diff
@@ -189,6 +189,8 @@ class BitmovinVideoPlayerPlugin : VideoPlayerPlugin {
         videoId: String?,
         viewerId: String?
     ) {
+        if (playerView == null || playerView?.player == null) return
```
[the remark this answers is not in the document]
Comment (KharchenkoAlex) [reply playback-sdk-android/src/main/java/com/streamamg/player/plugin/bitmovin/BitmovinVideoPlayerPlugin.kt:192]: I faced case when playerView created but the player was null inside player view

[the remark this answers is not in the document]
Comment (KharchenkoAlex) [reply playback-sdk-android/src/main/java/com/streamamg/player/plugin/bitmovin/BitmovinVideoPlayerPlugin.kt]: And yes, Kotlin support nullable chaining. Updated code

**Pull request merged**